src/components/hero.jsx

```diff
@@ -49,8 +49,8 @@ const Hero = () => {
 
       <Form />
 
-      <section className="block justify-center  items-center md:flex">
-        <div className="md:w-1/2 md:pr-20 md:text-left text-center">
+      <section className="block justify-center  items-center md:flex md:flex-col-reverse xl:flex xl:flex-row lg:flex lg:flex-row">
+        <div className="lg:w-1/2 xl:w-1/2 px-10 pt-5 lg:px-0 xl:px-0 lg:pr-20 lg:text-left md:text-left text-center">
```

Comment:
I think there is a bit of redundancy here. What do you think?
```suggestion
        <div className="lg:w-1/2 xl:w-1/2 px-10 pt-5 lg:px-0 xl:px-0 lg:pr-20 md:text-left text-center">
```

Reply:
I'll work on it